lib/db_utils.pm

```diff
@@ -232,3 +232,13 @@ sub check_postgres_db {
     }
     return 1;
 }
+
+is_ok_url {
+    # url connectivity check
+    my ($url) = @_;
+    return unless ($url and script_run('curl --help>/dev/null') == 0);
```

Comment:
I don't really see the need for this check, since `script_run` will return 0 if curl is available and there's an URL.  I see too much defensive programming here.

If we drop this check we're left with the `script_run`, thus making this function redundant.

But if you absolutely believe that we _need_ this function, let's:
1. Let's make the return value explicit rather than implicit.
2. Instead of running a command which forks a new process we can check that curl is available with a single `script_run('type -P curl')`

```suggestion
    return 0 unless ($url and script_run('type -P curl') == 0);
```